lya_sampler/py/emcee_sampler.py

```diff
@@ -36,6 +36,12 @@ def __init__(self,like=None,emulator=None,free_parameters=None,
         """Setup sampler from likelihood, or use default.
             If read_chain_file is provided, read pre-computed chain."""
 
+        # WE SHOULD DOCUMENT BETTER THE OPTIONAL INPUTS
+        # WHEN WOULD SOMEONE PASS A LIKELIHOOD AND A LIST OF FREE PARAMETERS?
+        # WOULDN'T like.free_params ALREADY CONTAIN THAT?
+
+        # WHEN WOULD YOU LIKE TO HAVE A SAMPLER WITHOUT AN EMULATOR?
```

Comment:
Similarly, we should also document this better. It will evolve in the next few days, so I'm ok leaving it like this for now.